src/components/ui/Pagination/Pagination.tsx

```diff
@@ -40,7 +40,7 @@ const Pagination = ({
         <ArrowLeft />
```

Comment:
Always mark the Icon const with `Icon` suffix. It will help recognize SVG code as a const.